src/popup/sass/layout/_tooltip.scss

```diff
@@ -0,0 +1,61 @@
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltiptext {
+  visibility: hidden;
+  position: absolute;
+  width: 180px;
+  background-color: #fff;
+  border: 2px solid #d8d8d8;
+  z-index: 400;
+  width: 20rem;
+  box-shadow: 10px 10px 10px -10px rgba(0, 0, 0, 0.25);
+  color: #333;
+  text-align: left;
+  padding: 5px;
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+
+.question-color {
+  color: #d8d8d8 !important;
+}
+
+.tooltip-right {
+  top: -5px;
+  left: 125%;
+}
+
+.tooltip-right::after {
+  content: '';
+  position: absolute;
+  top: 10%;
+  right: 100%;
+  margin-top: -5px;
+  border-width: 13px;
+  border-style: solid;
+  border-color: transparent #d8d8d8 transparent transparent;
+}
+
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+  opacity: 1;
+}
+
+.is-pulled-right {
+  float: right !important;
+}
+
+.caption,
+.image .image-title {
+  font-size: 0.8rem;
+}
+
+.margin-small {
+  margin: 0.1rem !important;
+}
+.padding-small {
+  padding: 0.5rem !important;
+}
```

Comment:
Please remove this and refer the [vocabulary spacing docs](https://cc-vocabulary.netlify.app/?path=/docs/tokens-spacing--helpers).

Reply:
Remove the css classes or the hardcoded values?

Reply:
remove these classes as they are already present in vocabulary.